config.yaml

```diff
@@ -52,8 +52,17 @@ uenvs:
         gh200: v0.9/gh200
       deploy:
         eiger: [zen2]
-        santis: [gh200]
         todi: [gh200]
+  editors:
+    "24.7":
+      recipes:
+        zen2: "24.7"
+        zen3: "24.7"
+        gh200: "24.7"
```

Comment:
What is the meaning of the version?

Reply:
July 2024.
Putting the date that it was configured, for future reference.

Reply:
I did not think of that... =)